app/models/user.rb

```diff
@@ -1,5 +1,6 @@
 class User < ApplicationRecord
   include GDS::SSO::User
+  has_paper_trail only: %i[role organisation_id has_access]
```

Comment:
Is this too conservative? This is either a sensible precaution not to log too much information or might be my fear of the unknown.

Reply:
I think this seems reasonable to me...Would we care about the history of the other attributes?